src/test/java/teammates/test/cases/action/StudentFeedbackSubmissionEditSaveActionTest.java

```diff
@@ -695,6 +703,97 @@ public void testExecuteAndPostProcess() throws Exception {
                 r.getDestinationWithParams());
         assertNotNull(frDb.getFeedbackResponse(fq.getId(), fr.giver, fr.recipient));
 
+        ______TS("Unsuccessful case: const sum: over maximum points");
```

Comment:
I agree that the `testExecuteAndPostProcess` function is super bloated. But if I create a function just for this, it might be suboptimizing. Let's let @tshradheya look at this in a more 'offline' manner during the summer.